### DISCUSSION TITLE
Fees and Other Payments

### DISCUSSION THREAD

**Discussion by ISGegalvin** [USEPA/supply-chain-factors#23] [Q&A] [answered]
Hi all,

I've been using companies' spend cubes to map to the EEIO emissions factors. I've read through this discussion board that taxes are already included in commodity spend. However, I also see many fees and other payments (e.g. taxes, permits, interest and debt, dividends, charitable contributions) in the company's spend. Do these payments have associated emissions factors? Should they be included in spend-based emissions inventories?

Thanks in advance for the help!

**Comment by WesIngwersen** [comment, marked as answer]
This would be specified in the standards or guidance used in emissions reporting, commonly the GHG Protocol. We do not claim to be authorities on any particular reporting standard. 

The supply chain factors just being a resource to help with emissions calculation, we have helped to answer questions about the use of the factors in relation to certain reporting categories defined in the [GHG Protocol Scope 3 guidance](https://ghgprotocol.org/scope-3-technical-calculation-guidance) such as #13 and #27.
